specs/DesignDiscussions/InputLabelsOptionLabel.md

```diff
@@ -0,0 +1,440 @@
+# Input/Label Associations
+
+This document covers design considerations related to issue [#203](https://github.com/microsoft/AdaptiveCards/issues/203).
+
```

Comment:
@almedina-ms / @dclaux / @matthidinger / @paulcam206 : Why am i already seeing labels /requires both referenced as part of Inputs already here in our [public documentation](https://adaptivecards.io/explorer/Input.Number.html)?